contracts/lib.rs

```diff
@@ -2275,7 +2267,7 @@ pub mod prosopo {
             let dapp_account = AccountId::from([0x2; 32]);
             // initialise the contract
             let mut contract = Prosopo::default(operator_account, STAKE_DEFAULT, STAKE_DEFAULT);
-            assert_eq!(0, contract.get_dapp_balance(dapp_account));
+            contract.get_dapp_balance(dapp_account).unwrap_err();
```

Comment:
Should this be returning `DappDoesNotExist` ?

Reply:
Same as above